docs/document/docs/guides/tabs-customzation.md

```diff
@@ -50,12 +52,54 @@ const salesOrderTabFields = [SALES_ORDER_ID];
 let currentTabFields;
 ```
 
-### Initialize Custom Component Tabs
+### Create some common functions
 
-We utilize the ReadOnlyTable component from the Kintone UI Component library to create the tab content.
-We create three ReadOnlyTable components: Sales Records, Task Management, and Sales Orders.
-We also define a `createContent()` function that takes an array of components and adds them to a container with inner padding.
-Finally, we call the `createTabs()` function to create the tabs and add them to the specified space.
+We create two common functions
+- Function `createContent()` to create a KUC Tabs. And for preventing DOM manipulations, we created the `isborderVisible` option.
+You can set Kintone native fields to Tabs content without DOM manipulation setting the `isborderVisible: false` option.
+- Function `addTabsToDom` to append the Tabs to the specified space.
+
+```javascript
+//You can set Kintone native fields to Tabs content without DOM manipulation setting the borderVisible: false option.
+function createTabs(contents, isBorderVisible) {
+  const items = [
+    {
+      label: 'Sales Record',
+      value: 'salesRecordTab',
+      content: contents ? contents[0] : ''
+    },
+    {
+      label: 'Task Management',
+      value: 'taskManagementTab',
+      content: contents ? contents[1] : ''
+    },
+    {
+      label: 'Sales Order',
+      value: 'salesOrderTab',
+      content: contents ? contents[2] : ''
+    }
+  ];
+  const tabs = new Kuc.Tabs({
+    value: 'salesRecordTab',
+    borderVisible: isBorderVisible,
+    items
+  });
+  return tabs;
+}
+
+function addTabsToDOM(tabs, space_id) {
+  const spaceElement = kintone.app.record.getSpaceElement(space_id);
+  spaceElement.appendChild(tabs);
+}
+```
+
+
+### Initialize Tabs for custom component
+
```

Comment:
In the native kintone field pattern explanation, there are these lines.
But for custom component, you just start from the content component explanation.
Please arrange the sentences to be able to understand what can be done with this scenario.
> We use the Tabs component feature to display different Kintone native fields depending on the tab.<br>
> When a tab is clicked, we show the corresponding Kintone native fields and hide the rest of the fields.<br>